api/swagger.yaml

```diff
@@ -1923,6 +1923,38 @@ definitions:
                 type: "array"
                 items:
                   type: "string"
+          Secrets:
+            description: "Secrets contains references to zero or more secrets that will be exposed to the service."
+            type: "array"
+            items:
+              type: "object"
+              properties:
+                File:
```

Comment:
This should be `Target` IMO.  File is misleading as users might think it's a full path but it's just the flat target under `/run/secrets`.